scripts/core/game.gd

```diff
@@ -34,16 +39,15 @@ func finalize_game() -> void:
 
 # Here we determine the winner of each result phase, runs before reseting the gameloop
 func determine_win() -> void:
-	var timeout = 3  # Default timeout for a loss
 	# If we have a match, we obviously win
 	if selected_card_a.card_data.name == selected_card_b.card_data.name:
-		timeout = 1  # Reduce timer to keep the player engadget
+		round_timeout = 1  # Reduce timer to keep the player engadget
```

Comment:
``round_timeline`` is overwritten here, forgetting the value that was set at ``@export``

Reply:
Yeah I thought, when winning we always want it to be 1 second. When losing, we can add some animation or whatever which might take a lil longer then 1 sec. This is why that variable was introduced, and overwritten as intended.

Perhaps we don't want an animation or effects to play when you lose, in that case we can perhaps just remove that part and hard code it to 1 second

Reply:
We could export both of the times:
```
@export_group("Round Settings")
@export var timeout_time: float = 1.5  # Default timeout for a loss
@export var correct_time: float = 1  # Default timeout for a win
```

As before, we use timeout by default (start of the function):
```
wait_time = timeout_time
```

Then in the if statement, we change it to correct time:
```suggestion
		wait_time = correct_time  # Reduce timer to keep the player engadget
```
And when changing time of the timer:
```
timer.wait_time = wait_time
```

Reply:
Also grammatical error in the comment ``engadget -> engaged``

Reply:
> We could export both of the times:
> ...

in case you wanna look at it zoomed out: 

```gdscript
@export_group("Round Settings")
@export var timeout_time: float = 1.5  # Default timeout for a loss
@export var correct_time: float = 1  # Default timeout for a win

...

func determine_win() -> void:
	wait_time = timeout_time
	# If we have a match, we obviously win
	if selected_card_a.card_data.name == selected_card_b.card_data.name:
		wait_time = correct_time  # Reduce timer to keep the player engadget
		print('WINNER, TODO HERE :eyes:')
		# Destroy cards? TODO
		finalize_game()  # Check if the board has been cleared.

	var timer = $setTimeout  # Obtain timeout component
	timer.wait_time = round_timeout  # Connect the finalize method to the timeout
	timer.start()  # Start the timeout counting down to execute the method hold by the timer

...
```

Reply:
Ye I like that approach more, agreed

Reply:
Sure I'll update it, but we should think about presenting "art people" options like that. Win scenario and/or lose scenario are both scenarios which have nothing to do with the current properties (based on what you provide).

What I try to say, is that certain options in the property window might throw a user off. Same was with card_prefab. It threw me off, so it would've had thrown off other people too.

I understand that @Visssarion wants it because debug purpose, we can always make it private later anyways thats no issue,